.github/workflows/deploy-aws.yml

```
@@ -0,0 +1,83 @@
+        uses: docker/build-push-action@v6
+        id: push
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Set digest output
+        id: digest
+        run: echo "digest=${{ steps.push.outputs.digest }}" >> $GITHUB_OUTPUT
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:1:51: Double quote to prevent globbing and word splitting

(shellcheck)